rocAL_pybind/rocal_pybind.cpp

```diff
@@ -476,6 +482,8 @@ PYBIND11_MODULE(rocal_pybind, m) {
     m.def("getTimingInfo", &rocalGetTimingInfo);
     m.def("labelReader", &rocalCreateLabelReader, py::return_value_policy::reference);
     m.def("cocoReader", &rocalCreateCOCOReader, py::return_value_policy::reference);
+    m.def("getRemainingImages", &rocalGetRemainingImages, py::return_value_policy::reference);
```

Comment:
There's already a pybind for this func. Any reason this is added?

Reply:
removing it